src/shared-components/carousel/arrow/style.js

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+
+import { SPACING, COLORS, ANIMATION } from 'src/constants';
+
+const ArrowContainer = styled.div`
+  cursor: pointer;
+  position: absolute;
+  top: 50%;
+  z-index: 2000;
```

Comment:
could we try to use one of the z-index constants here?

Reply:
you know what I actually checked the zScale constant before and I thought z-index:20 may not be sufficient ?? the other next value is modal:9999 which is a no go.
Should we create a new value or leave at 20?

Reply:
I guess we can create a new constant for now so that we at least have it in 1 place. Then if we ever straighten these z-index values out, it'll be easy for us to set it up

Reply:
done